### PR TITLE
 pre-requirement to make istio/envoy ci test happen in CI. 

### PR DESCRIPTION
**Pull request by innerpeacez**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [X] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: install/kubernetes/helm/skywalking/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v2
+name: skywalking
+home: https://skywalking.apache.org
+version: 0.1.2
+appVersion: 6.4.0
```
Comment (wu-sheng) [inline install/kubernetes/helm/skywalking/Chart.yaml]: Is this SkyWalking version? How we could keep this up to date with version change?

File: install/kubernetes/helm/skywalking/README.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+    ## Skywalking ui server Ingress TLS configuration
+    ## Secrets must be manually created in the namespace
+    ##
+    tls:
+      - secretName: skywalking-tls
+        hosts:
+          - skywalking.domain.com
+```
+### Envoy ALS
```
Comment (wu-sheng) [inline install/kubernetes/helm/skywalking/README.md]: This should be linked in https://github.com/apache/skywalking/blob/master/docs/en/setup/envoy/als_setting.md doc too.

**Comment by wu-sheng** [comment]
The old k8s doc is updated after our official release, how this works? Is it working with source code? If so, where is the process to prepare the docker image?

I am feeling the document is missing somehow.

**Comment by wu-sheng** [comment]
@hanahmily  And with this is going to be merged, what is the agenda for skywalking-kubernetes repo? Is that repo being abandoned? Or?

**Comment by innerpeacez** [comment]
> https://github.com/apache/skywalking/pull/3682#discussion_r337356044

> https://github.com/apache/skywalking/pull/3682#discussion_r337356506

resolved @wu-sheng

**Comment by hanahmily** [comment]
Our plan is to drop skywalking-kubernetes after an official release with kubernetes install relevant items in it.

Hongtao Gao

Apache SkyWalking && Apache ShardingSphere
Twitter, @hanahmily

**Comment by wu-sheng** [comment]
I mean the process, before merge this, I think we at least should have suitable process for release and heln update. Could we add the process to release doc?

**Comment by wu-sheng** [comment]
Otherwise, I have concerns. Release begin maybe in two weeks later. So, we need a clear purpose of this.

**Comment by kezhenxu94** [comment]
@innerpeacez can you re-title this PR? It's too vague, note that it will become the commit message in our codebase commit history

**Comment by hanahmily** [comment]
Sure, let me clarify it more clear. I don’t intend to finish this task before the next release. The process should be like the following:
1. make helm yaml work with istio and envoy
2. join it into CI workflow.
3. pick up a proper date to release it

So I expect that we can release it in 6.6 or 6.7. There’s a big gap between the current state and release ready.

**Comment by wu-sheng** [comment]
Do you two need to collaborate in upstream code base? Or just a fork repo is good enough?
Your process is clear to me. But merge into master and release of 6.5, 6.6 are still my concern.

**Comment by hanahmily** [comment]
I prefer to upstream/master branch. we need latest code base to verify the process, especially in the CI step.

and I know your concerns about source release, and my suggestion is to add `install` folder to “excluded” list in` create_source_release.sh`.

Hongtao Gao

Apache SkyWalking && Apache ShardingSphere
Twitter, @hanahmily

**Comment by wu-sheng** [comment]
But it will still misguide user. I want to offer a branch, istio-ci, which could use jenkins and our infra too, and don't need to be concernd about release issue.
Make sense?

**Comment by hanahmily** [comment]
Make sense to me. 

@innerpeacez what you should do is:

1. close current PR
2. send a PR to branch named `install/kubernetes`
3. follow suggestion in this PR, just like "Apache license header for each files” and etc.

Thanks a lot.

Hongtao Gao

Apache SkyWalking && Apache ShardingSphere
Twitter, @hanahmily

**Comment by innerpeacez** [comment]
Does the branch name need to be discussed? `install/kubernetes` or `istio-ci` ? @wu-sheng @hanahmily

**Comment by wu-sheng** [comment]
We are talking about branch. Not folder name.

**Comment by wu-sheng** [comment]
We should use istio-ci. As this is a primary goal as step1.

**Comment by wu-sheng** [comment]
Closing this. Branch `istio-ci` added, please submit the PR to there. https://github.com/apache/skywalking/tree/istio-ci, and tag the issue as `ci-setting` in milestone 6.6.

**Comment by hanahmily** [comment]
Any branch name is good to me. but the goal of step1 is to make helm work with istio mixer and envoy ALS. 
@innerpeacez don’t try to include any CI works in step1, because slim kubernetes cluster(minikube...) may 
block your step. I suggest do all of related work in a hosted kubernetes cluster, for example, on-premise,
AliCloud Kubernetes, EKS(AWS) and etc. 
I will send to issue to direct you how make it happen.

**Comment by innerpeacez** [comment]
Ok, I will submit it to the istio-ci branch soon. thanks a lot. @wu-sheng @hanahmily

**Pull request closed without merging**